v2/test/layer.spec.ts

```diff
@@ -236,6 +236,62 @@ describe("applyLayers", () => {
     expect(errors.length).toEqual(0);
   });
 
+  it("doesn't add layer to container image Lambda without extension or layer versions", () => {
+    const app = new App();
+    const stack = new Stack(app, "stack", {
+      env: {
+        region: "us-west-2",
+      },
+    });
+    const hello = new lambda.DockerImageFunction(stack, "HelloHandler", {
+      functionName: "container-lambda",
+      code: lambda.DockerImageCode.fromImageAsset("./test/assets"),
```

Comment:
nit: does this default to `Dockerfile` if a dir is specified? It might be a little more clear to explicitly specify the Dockerfile?

Reply:
Yep it's the dir of the Dockerfile. I just tried out specifying the Dockerfile itself and got this error, it has to be the dir: `Cannot find file at /Users/stephen.firrincieli/dev/datadog-cdk-constructs/v2/test/assets/Dockerfile/Dockerfile`

Reply:
ah okay interesting. I think that's okay for now. maybe if we decide to add future Dockerfiles at some point we can revisit.